sensor_msgs/PointCloud2 cloud
string offset 
---
bool result
